index.html

```diff
@@ -0,0 +1,1942 @@
+						  <a href="#">sponsored insight</a>
+						</div>
+						<div class="image">
+						  <a href="#"><span id="special-sub-2"></span>
+						  </a>
+						</div>
+					  </div>
+					  <div class="article-summary">
+						<h3 class="header-3">
+						  <a href="#">Think All International Schools are the Same? Think Again.</a>
+						</h3>
+						International schools are not created equal and each one possesses a unique blend of qualities and
+						educational specializations. Find out which one is best for your child.
+					  </div>
+					</article>
+				  </div>
+	
+				</div>
+			  </div>
+			  <div class="special-item-sub special-item-right">
+				<div class="content">
+				  <div class="head-story">
+					<article>
+					  <div class="article-wrap">
+						<div class="image">
+						  <a href="#"><span id="special-sub-3"></span>
+						  </a>
+						</div>
+					  </div>
+					  <div class="article-summary">
+						<h3 class="header-3">
+						  <a href="#">Think All International Schools are the Same? Think Again.</a>
+						</h3>
+						International schools are not created equal and each one possesses a unique blend of qualities and
+						educational specializations. Find out which one is best for your child.
+					  </div>
+					</article>
+				  </div>
+	
+				</div>
+			  </div>
+			  <div class="special-item-sub special-item-right">
+				<div class="content">
+				  <div class="head-story">
+					<article>
+					  <div class="article-wrap">
+						<div class="category">
+						  <a href="#">sponsored insight</a>
+						</div>
+						<div class="image">
+						  <a href="#"><span id="special-sub-4"></span>
+						  </a>
+						</div>
+					  </div>
+					  <div class="article-summary">
+						<h3 class="header-3">
+						  <a href="#">World's Leading Reputation Management Company 2020</a>
+						</h3>
+						NetReputation is the industry leader with an ever-growing scope of strategies and solutions to be
+						the
+						best all in one online reputation management services provider.
+					  </div>
+					</article>
+				  </div>
+	
+				</div>
+			  </div>
+			</div>
+	
+		  </div>
+		</div>
+		<!-- After page wrap closing tag -->
+		<div class="page-bottom ">
+		  <div class="inMagazine-block">
+			<div class="content">
+			  <div class="magazine">
+				<div class="full-row row align-items-end">
+				  <div class="magazine-phisical col-sm-3">
+					<div class="wrap">
+					  <div class="image">
+						<a href="#">
+						  <img class="img-fluid" src="images/january-03-17-2020-cover.jpg" alt="">
+						</a>
+					  </div>
+					  <div class="category">
+						<a href="#">January 17 issue</a>
+					  </div>
+					</div>
+					<a href="#" class="mgz-features">See all features</a>
+				  </div>
+				  <div class="magazine-3-features col-sm-9">
+					<div class="features-title box-title d-flex justify-content-center align-items-center">
+					  <span class="icon-news"></span>
+					  in the magazine
+					</div>
```

Comment:
- [ ] Check the alignment of the title. It should be centered.

- [ ] You are missing the N in the red square in all the sections.

Reply:
Added